data/adjacent_cubes_from_dolfin.py

```diff
@@ -3,30 +3,32 @@
 
 def make_adjacent_cubes_mesh(N, hdf5_filename):
     mesh = d.UnitCubeMesh(N, N, N)
-    mesh.coordinates()[:, 0] = mesh.coordinates()[:, 0]*2 - 1
-    mesh.coordinates()[:, 1] = mesh.coordinates()[:, 1]*2 - 1
-    mesh.coordinates()[:, 2] = mesh.coordinates()[:, 2]*4 - 2
+    mesh.coordinates()[:, 0] = mesh.coordinates()[:, 0] * 2 - 1
+    mesh.coordinates()[:, 1] = mesh.coordinates()[:, 1] * 2 - 1
+    mesh.coordinates()[:, 2] = mesh.coordinates()[:, 2] * 4 - 2
 
-    mf2 = d.MeshFunction('size_t', mesh, 2, value=0)
-    mf3 = d.MeshFunction('size_t', mesh, 3, value=0)
-    d.CompiledSubDomain('on_boundary').mark(mf2, 2)
-    d.CompiledSubDomain('on_boundary && near(x[2], 2)').mark(
-        mf2, 0)     # mark some facets near pm as 0
-    d.CompiledSubDomain('near(x[2], 0)').mark(mf2, 4)
-    d.CompiledSubDomain('x[2] <= 0').mark(mf3, 12)
-    d.CompiledSubDomain('x[2] >= 0').mark(mf3, 11)
+    mf2 = d.MeshFunction("size_t", mesh, 2, value=0)
+    mf3 = d.MeshFunction("size_t", mesh, 3, value=0)
+    d.CompiledSubDomain("on_boundary").mark(mf2, 2)
+    d.CompiledSubDomain("on_boundary && near(x[2], 2)").mark(

```

Comment:
Formatter splits into too many lines maybe?

Reply:
I believe the reason it is doing that in this particular case is because of the comment at the end of the line. One way to make to formatter put it on only one line would be to add the comment before the statement instead.